src/heroes/sombra/hack.opy

```diff
@@ -13,3 +13,16 @@ rule "[sombra/hack.opy]: Apply OW1 hack on top of OW2 hack":
     eventPlayer.setStatusEffect(eventPlayer.enemy_sombra_player, Status.HACKED, OW1_SOMBRA_HACK_ENEMY_DURATION)
     eventPlayer.hack_timer = OW1_SOMBRA_HACK_ENEMY_DURATION
     chase(eventPlayer.hack_timer, 0, rate=1, ChaseReeval.NONE)
+
+rule "[sombra/hack.opy]: Reset hack if has stealth passive":
+    @Event eachPlayer
+    @Hero sombra
+    @Condition eventPlayer.isFiringSecondaryFire()
+
+    wait(OW2_SOMBRA_HACK_CAST_TIME, Wait.ABORT_WHEN_FALSE)
+    eventPlayer.disallowButton(Button.SECONDARY_FIRE)
+    eventPlayer.allowButton(Button.SECONDARY_FIRE)
+    damage(eventPlayer, null, 0.01)
+    heal(eventPlayer, null, 0.01)
```

Comment:
```suggestion
    damage(eventPlayer, null, MIN_DAMAGE)
    heal(eventPlayer, null, MIN_DAMAGE)
```

Reply:
damaging 0.01 will result in sombra grunting when taking damage

Reply:
i suppose so but i think this should be the only way resetting it or something but yeah  i think i could also find another way to fix this too. other than not using damages.

Reply:
i guess for now this is an okay solution???

Reply:
MIN_DAMAGE is equivalent to 0.00001 damage or something

it should function the same as applying 0.01 damage, but with the added benefit of not making the character to grunt